microsoft-edge/progressive-web-apps-chromium/how-to/service-workers.md

```diff
@@ -10,13 +10,13 @@ ms.date: 11/28/2022
 ---
 # Use service workers to manage network requests
```

Comment:
make title singular?  
Use a service worker to manage network requests

singular is often clearer.  one's own PWA would use a single s.w.?  if so, the task, literally, is to create a SINGLE s.w. -- whereas the abstract topic of discussion is plural, "about s.w.s"
Prefer task (singular i think here), rather than framing as abstract discussion of plural Foos.

@captainbrosset - make TOC verbose & match top-of-article title?  

rendered article in PR: 
https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/service-workers?branch=pr-en-us-3215
![image](https://github.com/MicrosoftDocs/edge-developer/assets/45407486/3423dd0f-a903-4309-b417-dcd8dc4e118d)

Reply:
changed intro sentence to consistently singular - as typical, result is clearer:
```
A _service worker_ is a special type of [web worker](https://developer.mozilla.org/docs/Web/API/Web_Workers_API) 
that's able to intercept, modify, and respond to network 
requests by using the `Fetch` API.  Service workers can 
access the `Cache` API, and asynchronous client-side data 
stores, such as `IndexedDB`, to store resources.
```

Reply:
example of confusion about plural, to examine throughout article:
"service workers must exist in a separate file"
Seems contradictory and technically incorrect.  1 file contains multiple sw's?  Favoring singular gets rid of such many-to-1 ambiguity.

Reply:
I'm on board with using singular here. Feel free to make the change globally.
Also, 1 file contains just 1 service worker.

Reply:
I'll move article from plural toward singular, and might explicitly state 1 file = 1 sw.

Reply:
changed title to singular, globally in repo

Reply:
```
Similar to other web workers, a service worker must exist in a 
separate file.  The file contains a single service worker.  You 
reference this file when registering the service worker, as shown 
in ...
```

Reply:
changed to singular s.w. in this file